aiographite/aiographitesend.py

```diff
@@ -0,0 +1,449 @@
+import asyncio
+from aiographite.graphite_escaping import metrics_name_to_graphite, metrics_name_from_graphite
+import pickle
+import struct
+import socket
+import os
+import time
+
+
+DEFAULT_GRAPHITE_PLAINTEXT_PORT = 2003
+DEFAULT_GRAPHITE_PICKLE_PORT = 2004
+SUPPORT_PROTOCOLS = ["plaintext", "pickle"]
+
+
+class AioGraphiteSendException(Exception):
+    pass
+
+
+class AsyncioGraphiteSendService(object):
+
+	def __init__(self, graphite_server, graphite_port, protocol = "pickle"):
+
+		self.graphite_server_address = (graphite_server, graphite_port)
+
+		self.connect_to_graphite()
+
+		self.protocol = protocol
+
+		self.loop = asyncio.get_event_loop()
```

Comment:
for the event loop, it would be nice allow someone to pass the event loop as well, kind of like aiohttp:

http://aiohttp.readthedocs.io/en/stable/#getting-started

We can default to asyncio.get_event_loop() though, if one is not passed.